src/store.js

```diff
@@ -22,13 +23,13 @@ export default () => {
 function saveState(state) {
 	let saved = {};
 	for (let i=SAVE.length; i--; ) saved[SAVE[i]] = state[SAVE[i]];
-	localStorage.state = JSON.stringify(saved);
+	localStorageSet('state', JSON.stringify(saved));
 }
 
 function getSavedState() {
 	let state;
 	try {
-		state = JSON.parse(localStorage.state);
+		state = JSON.parse(localStorageGet('state'));
```

Comment:
This line is already wrapped in a try-catch, so it was safe already, but i’ve used localStorageGet here for consistency.